internal/driver/nodeserver.go

```diff
@@ -86,15 +88,20 @@ func (ns *NodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 	log, _, done := logger.GetLogger(ctx).WithMethod("NodePublishVolume")
 	defer done()
 
+	functionStartTime := time.Now()
 	volumeID := req.GetVolumeId()
 	log.V(2).Info("Processing request", "volumeID", volumeID)
 
 	ns.mux.Lock()
 	defer ns.mux.Unlock()
 
+	functionStatus := metrics.Completed
```

Comment:
Do we need to have this `functionStatus` var at all? Why don't we just pass the value directly to `RecordMetrics()`?

Reply:
Yeah! That should work!